pkg/project/util.go

```diff
@@ -175,3 +177,14 @@ var intersection = apply(func(bs []Equatable, a Equatable) bool {
 var difference = apply(func(bs []Equatable, a Equatable) bool {
 	return !contains(bs, a)
 })
+
+var getEnvIntVar = func(key string, fallback int) int {
+	value, exists := os.LookupEnv(key)
+	if exists {
+		intValue, err := strconv.Atoi(value)
+		if err == nil {
```

Comment:
If we can't convert to integer, then we should return a warning to the users so they are aware that the default is used instead.

Reply:
warning print on the console using 
```
value, err := getEnvIntVar("REPO_LIMIT_OVERRIDE", defaultMaxItems)
					if err != nil {
						fmt.Println(err)
					}
```
please suggest if there is better way.